% !TEX program = latexmk -xelatex
\documentclass[11pt,a4paper,roman]{moderncv}
\moderncvstyle{banking}
\moderncvcolor{blue}
%\renewcommand{\familydefault}{\rmdefault}
%\nopagenumbers{}
\usepackage{verbatim}


% adjust the page margins
\usepackage[scale=0.75]{geometry}
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvtitlenamewidth}{10cm}           % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...


% font loading
% for luatex and xetex, do not use inputenc and fontenc
% see https://tex.stackexchange.com/a/496643
\ifxetexorluatex%
  \usepackage{fontspec}
  \usepackage{unicode-math}
  \defaultfontfeatures{Ligatures=TeX}
  \setmainfont{Latin Modern Roman}
  \setsansfont{Latin Modern Sans}
  \setmonofont{Latin Modern Mono}
  \setmathfont{Latin Modern Math}
\else
  \usepackage[utf8]{inputenc}
  \usepackage[T1]{fontenc}
  \usepackage{lmodern}
\fi

% personal data
\name{Gianluca}{Della Vedova}
\email{gianluca.dellavedova@unimib.it}
\homepage{https://www.unimib.it/gianluca-della-vedova}
\social[github]{gdv}
\social[orcid]{0000-0001-5584-3089}
%\social[googlescholar]{0gaIFokAAAAJ}


% to show numerical labels in the bibliography (default is to show no labels); only useful if you make citations in your resume
\makeatletter
\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
\makeatother
\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}
\usepackage[resetlabels]{multibib}
\newcites{inbook,article,conference,P}{{Book chapters},{Journal
articles},{Conference proceedings}, {Preprints}}

% \usepackage[style=numeric,minnames=8,maxnames=8,defernumbers=true,sorting=ydnt]{biblatex}
% \addbibresource{gdv.bib}

% define different groups for references here
% \DeclareBibliographyCategory{conference}
% \DeclareBibliographyCategory{journal}

% customize format how bib sections are rendered, here: regular \section{}
% \defbibheading{mybibsection}[\bibname]{%
% \section{#1}%
% }


\title{CV}
\begin{document}
\makecvtitle%

\section{Current position}

\noindent
  Associate professor with tenure (SSD INF/01 --- Informatica)\\
  Dipartimento di Informatica, Sistemistica e Comunicazione\\
  Università degli Studi di Milano -- Bicocca

\section{Positions}

  \cventry{10/2012--today}{Dept. of Informatics, Systems, and Communications}{Associate Professor}{Univ. Milano -- Bicocca}{}{}
  \cventry{10/2005--09/2012}{School of Statistics}{Associate Professor}{Univ. Milano -- Bicocca}{}{}
  \cventry{05/2001--09/2005}{School of Statistics}{Assistant Professor}{Univ.Milano -- Bicocca}{}{}

\section{Habilitation}

  \cventry{08/2018--today}{Italian habilitation to be Full professor of Computer Science}{Abilitazione a Professore di I fascia, settore concorsuale 01/B1}{}{}{}

\section{Education}

  \cventry{1996--2001}{}{Ph.D. in Computer Science}{Università degli Studi di
  Milano}{}{Thesis: ``Multiple Sequence Alignment and Phylogenetic Reconstruction:
  Theory and Methods in Biological Data Analysis''}{}

  \cventry{1990--1995}{}{M.Sc. in Computer Science}{Università degli Studi di Milano}{}{Thesis: ``Sequential and Parallel Algorithms for Graph Decomposition''}{}

\section{Funding}
\subsection{Leading}


  \cvitem{2020--2023}{%
  H2020-MSCA-RISE-2019 (Univ. Milano -- Bicocca amount: 197 800€)
  Pan-genome Graph Algorithms and Data Integration (PANGAIA) ---
  Horizon 2020 Marie Skłodowska-Curie Research and Innovation Staff Exchange programme. In this project, whose coordinator is a member of my research lab, I have been one of the main participants in planning and writing the proposal. I am
  \textbf{leading WP5} Communication and Dissemination.
  The project requires the collaboration among 7 beneficiaries (4
  universities, 1 research center, and 2 enterprises) in 6 European countries and 4 partners in US, Canada, and Japan.
  }
  \cvitem{2021--2024}{%
  H2020-MSCA-ITN (Univ. Milano -- Bicocca amount: 261 499,68€) ALgorithms for PAngenome
  Computational Analysis (ALPACA) ---
  Horizon 2020 Marie Skłodowska-Curie Innovative Training Network. I am
  \textbf{co-supervising} a PhD student and a member of the Executive
    Committee.
  The project has 13 beneficiaries (7 universities, 5 research centers, 1
  company) and 10 european partners.
  }
  \cvitem{2013--2016}{%
  Modulation of anti-cancer immune response by
  regulatory non-coding RNAs --- Fondazione Cariplo 2013.
  I have been in \textbf{charge of the bioinformatics WP}. The project had two
  research units: the National Institute of Molecular Genetics and the Univ.
  Milano -- Bicocca.
  During this project, I have been the \textbf{scientific supervisor of two
  1-year post-docs} and I have co-supervised a third postdoc.
  }
  \cvitem{2011--2014}{%
  MIUR/Regione Lombardia 2011 (Univ. Milano -- Bicocca amount: 199 991€).
  Piattaforma di Analisi TRaslazionale Integrata (PATRI).
  I have been \textbf{in charge of all bioinformatics aspects}.
  This project has led to a 1-year postdoc (23 000€) and 3 short-term research
  contracts (each between 10 000€ and 12 000€).
  I have co-supervised the postdoc and I have been the only supervisors of all
  short-term contracts.
  }
  \cvitem{2016}{%
  Fondo di Ateneo 2016 (12 490€). Modelli computazionali e algoritmi:
  aspetti teorici e sperimentali, con applicazioni alla Bioinformatica.
  \textbf{PI}.
  }
  \cvitem{2015}{%
  Fondo di Ateneo 2015 (10 980€). Algoritmi combinatori e modelli di
  calcolo: aspetti teorici e applicazioni in Bioinformatica.
  \textbf{PI}.
  }
  \cvitem{2014}{%
  Fondo di Ateneo 2014 (12 186€). Algoritmi e modelli computazionali:
  aspetti teorici e applicazioni nelle scienze della vita.
  \textbf{PI}.
  }
  \cvitem{2013}{%
  Fondo di Ateneo 2013 (9 337€). Metodi algoritmici e modelli: aspetti
  teorici e applicazioni in bioinformatica. \textbf{PI}.
  }
  \cvitem{2011}{%
  Fondo di Ateneo 2011 (4 055€). Tecniche algoritmiche avanzate in
  Biologia Computazionale. \textbf{PI}.
  }
  \cvitem{2006}{%
  Grandi Attrezzature 2006 (40 000€). Laboratorio Virtuale
  Statistico-Territoriale --- Virtual Statistical--Territorial Laboratory.
  This was a joint project between the Dept. of Statistics and the Dept. of
  Sociology. It has resulted in the acquisition of 2 servers that are used to
  provide services to research and teaching activities on territorial sociology
  and computational statistics.
  \textbf{co-PI}.
  }
  \cvitem{2005--2008}{%
  MIUR/PRIN 2005, Potenzialità e ottimizzazione delle banche dati
  automatizzate in epidemiologia.
  \textbf{In charge of all bioinformatics aspects}.
  }

\subsection{As a Participant}


\cvitem{2013--2016}{%
  Regione Lombardia. SPAC3 --- Servizi smart della nuova Pubblica
  amministrazione per la Citizen-Centricity in cloud.}
  \cvitem{2011--2014}{%
  MIUR/PRIN 2011
  Automi e Linguaggi Formali: Aspetti Matematici e Applicativi}
  \cvitem{2003}{%
  MIUR/FIRB 2003. Bioinformatica per la Genomica e la Proteomica.}
  \cvitem{2000--2001}{%
  NSF CCR-9988353, ITR-0085910.}
  \cvitem{1999--2001}{%
  MURST COFIN 98 ``Bioinformatica e ricerca genomica''.}
  \cvitem{1994--1995}{%
  MURST 40\% ``Algoritmi e strutture di calcolo''.}
  \cvitem{1994--1995}{%
  ESPRIT-BRA ASMICS 2 n.~6317.
  }

\section{Research activity}


I have published more than 50 journal papers with more than 1800
citations and h-index 20 (from Google Scholar).
According to Google Scholar, 7 of my papers have at least 100 citations each, and I have 166 coauthors.

From 2016 to 2019 I have been the leader of the Research Lab
``AlgoLab --- Experimental Algorithmics Lab'', at the Università di Milano -- Bicocca.
The research lab has several international collaborations and it focuses on algorithm design, implementation, and experimental analysis on large datasets. In fact, the research activities regards both methodological and experimental aspects, leading to the development of software tools to attack bioinformatics problems.

My personal research activity has focused on the development of combinatorial
algorithms in Bioinformatics, with a strong attention on foundational aspects,
such as the formal properties of computational problems.
Moreover, I have paid special attention to algorithm implementation and validation, from both a theoretical and an experimental point of view.


This research field has gained relevance with the establishment of the
Human Genome project whose main goal is to determine the influence that
biological sequences (such as DNA, RNA) have on living beings.
Completing the sequencing of the human genome was only the first
fundamental step; finding which proteins are expressed by each gene and
determining the interaction among the various DNA sites are among the
most important open problems. The huge quantity of data that has to be
analyzed (remember that human DNA contains 3 billions nucleotides) makes
computers and efficient algorithms a needed cornerstone of the field.
My research has been mainly devoted to the design of such
efficient algorithms and can be detailed in a few subfields.

\subsection{Sequence comparison}\label{sequence-comparison}

The central dogma in Computational Biology states that sequence homology
leads to functional homologies (that is similarity among the effects
performed by such sequences), therefore it is of the utmost importance
to have some computational tools for comparing sequences. In this
direction the multiple sequence alignment problem has been formally
introduced in (Altschul, Lipman SIAM J. Appl. Math. 1989), even though
it had already been studied previously, and some of its variant are
intractable. I have studied when this problem can be solved efficiently,
even in an approximate sense. I have developed new algorithms for some
different notions of sequence comparison, related to the classical
notion of longest common subsequence and shortest common supersequence.
Those two combinatorial problems have found new applications in
Molecular Biology.

Most recently, I have started the investigation of the comparison of sequences
and labeled graphs representing entire genomes (pangenome graphs).
In this field I have focused on the development of efficient and practical
alignment algorithms that fully exploit the topology of the graphs.

\textbf{Journal
papers}:~\cite{baaijensComputationalGraphPangenomics2022,DBLP:journals/bioinformatics/DentiPPCVRB21,DBLP:journals/ipl/BonizzoniVDP10,DBLP:journals/tcbb/BonizzoniVDFRV07,DBLP:journals/informs/JustV04,DBLP:journals/dam/BonizzoniVM01,DBLP:journals/tcs/BonizzoniV01}

\subsection{Phylogeny reconstruction and
comparison}\label{phylogeny-reconstruction-and-comparison}

Reconstructing phylogenies is another problem that has great relevance
in Computational Biology, as a phylogeny is an intuitive representation
of a common evolutionary history of a set of extant species. In this
setting I have studied the quartet-based reconstruction technique (a
quartet is the optimal solution over four species), developing new
algorithms to clean some of the error that inherent in the use of such
technique.

The phylogeny comparison problem is fundamental when you have to compare
the results of various experiments on the same set of species. I have
extensively studied the problem In (Amir e Keselman, SIAM J. Comp. 1997)
some formulations of the problem have been introduced. The candidate has
studied the computational complexity of the problem, finding some
negative results on the approximability of the maximum common isomorphic
subtree, describing an L-reduction and then applying the
self-improvement technique (Karger, Motwani e Ramkumar, Algorithmica
1997). The candidate has also studied some other variants of the
phylogeny comparison problem, more precisely when dealing with trees
computed on different genes over the same species set, designing some
efficient algorithms and solving a problem left open in (Ma, Li e Zhang,
SIAM J. Comp. 2000).

\textbf{Journal
papers}:~\cite{DBLP:journals/jcb/AliCLVP21,DBLP:journals/bioinformatics/CiccolellaRGPSB21,DBLP:journals/bioinformatics/CiccolellaBDBPV21,DBLP:journals/bmcbi/CiccolellaGPVHB20,DBLP:journals/tcbb/BonizzoniCVS19,DBLP:journals/tcs/BonizzoniCVRT17,DBLP:journals/fuin/BonizzoniCVRT17,bonizzoniExplainingEvolutionConstrained2014,DBLP:journals/tcs/BonizzoniVD05,DBLP:journals/bioinformatics/VedovaW02,DBLP:journals/ijfcs/BonizzoniVM00}

\subsection{DNA microarray design}\label{dna-microarray-design}

The introduction of DNA microarray have greatly increased the throughput
of experimental data in Molecular Biology. Such technology (Drmanac et.
al.~Science 1991) has given relevance to some computational problems on
the optimal microarray synthesis or on the experimental data analysis.
More precisely, classifying microbial communities can be performed only
exploiting microarrays, as the microbial external aspect is hard to
study, due to their extremely small size. A fundamental computational
problem in this field is computing the minimum set of substrings that
are able to distinguish a set of strings (Probe Selection). The
importance of this problem is due to the fact that it formalizes the
search for the cheapest experiment obtaining the desired result. The
candidate has proposed, implemented and analyzed some algorithms, while
supervising the use of the implementation on some biological data that
were previously impossible to analyze. This effort has led to a new
protocol for microbial communities analysis.

\textbf{Journal
papers}:~\cite{DBLP:journals/bmcbi/RueggerVJB11,valinskyAnalysisBacterialCommunity2002,valinskyOligonucleotideFingerprintingRRNA2002,bornemanProbeSelectionAlgorithms2001}

\subsection{Haplotyping}\label{haplotyping}

Several species, including human beings, are diploid, that is each
chromosome consists of two distinct copies called haplotypes. Current
technological limitations do not allow to cheaply compute those
haplotypes, but only genotypes (that is the two nucleotides that are in
the same position in those haplotypes). Since it is important to know
the actual haplotypes, a number of related computational problems have
been recently introduced; those problems compose the field of
haplotyping.

I have designed and analyzed an algorithm to complete haplotypes on
incomplete data and under the coalescent model (such models forbids some
otherwise possible recombinations), where the objective function is the
entropy of the solution. The algorithm belongs to the class of
Kernighan-Lin heuristics and it has been empirically and favorably
compared to the greedy algorithm that was previously routinely employed.

Moreover I have studied the xor-genotyping problem, where the input data
contains even less information that in most other haplotyping problems,
as it is known only the positions where the two haplotypes differ, and
not their contents. This formulation is a faithful model of the results
that can be obtained with a recent and economically viable technology.
I have obtained some preliminary results on the computational complexity
of the problem, and he has designed some efficient algorithms.

\textbf{Journal papers}:~\cite{ciccolellaMALVIRUSIntegratedApplication2022,DBLP:journals/bmcbi/BerettaPZVB18,DBLP:journals/tcbb/PirolaVBSB12,DBLP:journals/tcbb/BonizzoniVDPR10,DBLP:journals/ijbra/BonizzoniVDM05,DBLP:journals/jcst/BonizzoniVDL03}

\subsection{Graph Algorithms}\label{graph-algorithms}

The second main area of research is designing efficient combinatorial
algorithms for some problems on graphs. Graph theory is one of the most
important research fields that are common to Computer Science and
Discrete Mathematics, as graphs are a mathematical device that is
suitable for natural modeling of various real-world problems. One of the
techniques that has been widely employed for designing efficient graph
algorithm consists of decomposing the graph and then solving the problem
on the smaller parts for finally recombining the partial solutions. The
candidate has studied a kind of decomposition called modular
decomposition, introduced by Gallai in 1967 for graphs and successively
extended to 2-structures (Ehrenfeucht e Rozenberg Theor. Comp. Sci.
1990) and k-structures (Ehrenfeucht e McConnell, Theor. Comp. Sci.
1993). I have developed some efficient algorithms for
computing the modular decomposition on hypergraphs and k-structures.


\textbf{Journal papers}:~\cite{DBLP:journals/jal/BonizzoniV99}


\subsection{Clustering}\label{clustering}

The problem of classifying data in similar sets is one of the most
important problems in Computer Science; it is common to have a
similarity measure between pairs of elements and to aim at computing a
partition of the elements so that elements in a common sets are similar
while elements in different sets are not similar. In this field
I have the recently introduced correlation clustering
problem (Bansal, Blum e Chawla, FOCS 2002) where a weighted graph is the
main tool for formally modeling the problem. I have also pointed out
some connections with another version of clustering, showing that an
interesting restriction of the problem is NP-complete and providing two
polynomial-time approximation schemes for a different formulation of the
problem.

Moreover I have studied a different clustering problem, closely related
to the analysis of microarray data. In this case, data are represented
as vectors on \(\{0,1,N\}\) alphabet, where \(N\) stands for missing or
undecided data. His contributions consist of proving that some
restrictions of the problem are APX-hard and designing a polynomial-time
constant-factor approximation algorithm. Moreover, I have designed an
efficient algorithm for a different restriction of the problem.

Another research topic related to Clustering is the \(k\)-anonymity problem,
where we want to cluster the rows of a matrix so that each cluster has identical
entries, except for \(k\) rows.
This problem has applications in data privacy.


\textbf{Journal papers}:~\cite{DBLP:journals/titb/CiccolellaPBV21,DBLP:journals/jco/BonizzoniVDP13,DBLP:journals/tcs/BonizzoniVD12,DBLP:journals/jco/BonizzoniVD11,DBLP:journals/algorithmica/BonizzoniVDM10,DBLP:journals/jcss/BonizzoniVDJ08}

\subsection{Alternative Splicing}\label{splicing}

Alternative splicing is the biological mechanism that allows a gene to encode
and produce more than one protein.
In this field I have focused on efficient algorithms for detecting novel (that
is not previously known) alternative splicing events.


\textbf{Journal papers}:~\cite{DBLP:journals/bmcbi/DentiRBVPB18,DBLP:journals/jcb/BerettaBVPR14,DBLP:journals/bmcbi/PirolaRPPVB12}

\subsection{Text Algorithms}\label{bwt}

The sheer size of genomic data makes important to develop data structures to
represent with a small amount of memory while querying them efficiently.
The main such data structure is the Burrows-Wheeler Transform (BWT), and I have
developed some algorithms to efficiently compute the BWT together with another
auxiliary data structure (the Longest Common Prefix array) and for building the
string graph (a data structure that represents how some strings overlap with
each other).

\textbf{Journal papers}:~\cite{DBLP:journals/tcs/BonizzoniVPPR21,DBLP:journals/jcb/BonizzoniVPPR19,DBLP:journals/quanbio/RizziBPPPVB19,DBLP:journals/algorithmica/BonizzoniVPPR17,DBLP:journals/jcb/BonizzoniVPPR17,DBLP:journals/jcb/BonizzoniVPPR16}


\section{Teaching}

I have started teaching when I have been hired as an Assistant professor at
  the School of Statistics in 2001.
  I was the first Computer Science faculty of the school, therefore I had to
  design new courses tailored to students with a good mathematical background,
  but with no Computer Science experience.
  During my career I had to design a course several times, partly due to the
  evolution of the background of the students and to changes in the overall
  degree in Statistics and Statistical Economics, partly because I had to teach
  also to students majoring in Computer Science or in Data Science.

  My teaching philosophy is based on active learning. This requires important
  and continuous interactions between teacher and students, requiring the
  development of problems that students have to attack, as a group or solo.
  Designing an active learning course is usually innovative, since existing
  courses are mostly designed for more traditional approaches, where students
  have a more passive role and acquiring problem solving skills is not a main focus.

  \subsection{Ph.D. courses taught}


\cvitem{2016, 2018, 2020}{Advanced Algorithms, PhD in Computer Science, Univ.
Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
}

\cvitem{2021}{Graph Theory and Algorithms, PhD in Computer Science, Univ.
Milano -- Bicocca.
  I have co-designed the course, since it had never been taught before.
}

\subsection{Courses taught --- principal teacher in charge of the
course}

\cvitem{2022--today}{%
    ``Large-Scale Graph Algorithms'',
    M.Sc. in Computer Science,
Univ. Milano -- Bicocca (6 ECTS).
  I have designed the course, since it had never been taught before.
}

\cvitem{2021--today}{%
    ``Laboratorio di Informatica (Computer Science Lab)'',
    B.Sc. in Statistics, B.Sc. in Statistical Economics,
Univ. Milano -- Bicocca (3 ECTS).
I had fully redesigned the course.
}


\cvitem{2017-today}{%
``Foundations of Computer Science'', M.Sc. in Data Science,
Univ. Milano -- Bicocca. (6 ECTS).
  I have designed the course, since it had never been taught before.
}
\cvitem{2014-today}{%
    ``Elementi di Bioinformatica (Elements of Bioinformatics)'', B.Sc. in Computer Science, Univ.
Milano -- Bicocca (8 ECTS).
  I have designed the course, since it had never been taught before.
  }
\cvitem{2001--2020}{%
    ``Basi di Dati (Databases)'', B.Sc. in Statistics, B.Sc. in Statistical Economics,
Univ. Milano -- Bicocca (6 ECTS).}

\cvitem{2007-2022}{%
    ``Bioinformatica (Bioinformatics)'', M.Sc. in Biostatistics, Univ.
Milano -- Bicocca (6 ECTS).
  I have designed the course, since it had never been taught before.
  }

\cvitem{2010--2013}{%
    ``Algoritmi su stringhe (Text Algorithms)'', B.Sc. in Computer Science, Univ.
Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }
\cvitem{2008}{%
    ``Strumenti informatici per la statistica M (Computational Tools for
    Statistics)'', M.Sc. in
Biostatistics, blended e-learning, Univ. Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }
\cvitem{2007--2009}{%
    ``Informatica Applicata S (Applied Computer Science)'', M.Sc. in Biostatistics (2
ECTS), Univ. Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }
\cvitem{2001--2008}{%
    ``Laboratorio Statistico-Informatico (Computational Statistics Lab)'', all B.Sc. programs of the School
of Statistics (6 ECTS), Univ. Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }
\cvitem{2001--2008}{%
    ``Programmazione e Basi Dati (Programming and Databases)'',
all B.Sc. programs of the School of Statistics
(6 ECTS), Univ. Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  The course has changed name to "Basi di Dati".
  It was the first database course in Italy designed for students in
  Statistics.
  }
\cvitem{2008}{%
    ``Fondamenti di Informatica (Fundamentals of Computer Science)'', M.Sc. in Biostatistics, Univ.
Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }
\cvitem{2006}{%
    ``Laboratorio di Informatica (Computer Science Lab)'',
all B.Sc. programs of the School of Statistics
(6 ECTS), Univ. Milano -- Bicocca.
  I have designed the course, since it had never been taught before.
  }

\subsection{Postgraduate courses}

\cvitem{2021}{%
    ``Beyond Genome Assembly'',
  Master   ``qOmics: quantitative methods for Omics Data'',
Univ.
Milano -- Bicocca and Univ. Pavia.}

\cvitem{2012, 2014}{%
    ``Data Base e Sistemi Informativi (Databases and Information
    Systems)'',
Master di primo livello in
Amministratore di Sistema per la Diagnostica per Immagini (Master in
Image Diagnostics Systems Administrator),
Univ.
Milano -- Bicocca.}

\cvitem{2007, 2010, 2011}{%
    ``Fondamenti di Informatica (Fundamentals of Computer Science)'',
Master di primo livello in
Amministratore di Sistema per la Diagnostica per Immagini (Master in
Image Diagnostics Systems Administrator),
Univ.
Milano -- Bicocca.}
\cvitem{2003}{%
``Fondamenti di Informatica e Elementi di Programmazione'' (Fundamentals
of Computer Science and Elements of Programming), Master di primo
livello in Bioinformatica
(Master in Bioinformatics), Univ. Milano -- Bicocca.}
\cvitem{2001--2002}{%
    ``Sistemi Informatici e Elementi di Programmazione (Introduction to
    Programming)'', Master di primo
livello in Bioinformatica (Master in Bioinformatics), Univ. Milano -- Bicocca.}

\section{Supervision}

\subsection{Postdocs}

I have been \textbf{scientific supervisor} of the following postdoctoral
positions:

\cvitem{2022--today}{%
Simone Ciccolella. 2-year postdoc position on ``Computational Approaches for
tumor and viral phylogenies''.
}

\cvitem{2018--2019}{%
Murray Patterson. 2-year postdoc position on ``Haplotype
assembly from sequencing reads''. Currently Assistant Professor
(Tenure-track) at Georgia State University.
}

\cvitem{2015}{%
Hassan Mahmoud Mohamed Ramadan Mohamed.
1-year postdoc position on
``Methodology for treatment and data analysis of NGS data for
the detection of alternate splicing events''
}

Moreover, I have contributed to the training of the following postdocs:

\cvlistitem{Luca Denti}
\cvlistitem{Marco Previtali}
\cvlistitem{Stefano Beretta}

\subsection{PhD students supervision}

\textbf{Supervisor} of


\cvitem{2021--2024 (expected)}{%
  Jorge Avila Cartes --- PhD in Computer Science}

\cvitem{2017--2021}{%
  Simone Ciccolella, ``Algorithms for cancer phylogeny inference'' --- PhD in
  Computer Science.
  Currently Post-doc at Università di Milano -- Bicocca.}

\cvitem{2013--2017}{%
  Marco Previtali, ``Self-indexing for de novo assembly'' --- PhD in Computer Science.
Currently at Bloomberg}

\cvitem{2008--2012}{%
  Stefano Beretta, ``Algorithms for Next Generation Sequencing Data
  Analysis'' --- PhD in Computer Science.
  Currently at San Raffaele Telethon Institute for Gene Therapy}

\cvitem{2001--2004}{%
  Riccardo Dondi, ``Computational Problems in the Study of Genomic
  Variations'' --- PhD in Computer Science.
  Currently Associate Professor at  Università di
  Bergamo}

Moreover, I have actively contributed to the training of the following
PhD students:

\cvlistitem{%
    Giulia Bernardini --- PhD in Computer Science.
    Currently Assistant Professor at Univ. Trieste.}
\cvlistitem{%
  Simone Zaccaria --- PhD in Computer Science.
  Currently   Group Leader presso Department of   Oncology, Univ. College London}
  \cvlistitem{%
  Anna Paola Carrieri --- PhD in Computer Science.
  Currently   Research Staff Member presso IBM
  Research Lab, UK}

\subsection{M.Sc. students supervision}

  I have supervised or co-supervised the thesis of more than 10 M.Sc.
  students  in Computer Science, Statistical Economics, Biostatistics,
  Data Science at University of Milano--Bicocca and at the University of Milan.
  I am currently supervising the thesis of a M.Sc. in Bioinformatics at the
Pwani University Biosciences Research Centre, Kilifi (Kenya).



  I have supervised the activities related to the
  \textbf{Exchange mobility EXTRA UE} program of:

  \cvlistitem{%
  Ramesh Rajaby ha spent 6 months with the research group led by
    prof. Jesper Jansson (Kyoto University, Japan). Ramesh Rajaby is
    currently postdoc at  National University of Singapore.}
    \cvlistitem{%
    Simone Ciccolella has spent 3 months with the research group led by
prof. Iman Hajirasouliha (Weill Cornell Medicine, New York).
Simone Ciccolella is currently a postdoc under my supervision.}

  \subsection{B.Sc. students supervision}

  I have supervised the final projects of more than 50 B.Sc. students in
  Computer Science, Statistics, and Statistical Economics.

\subsection{Incoming visits}

\cvlistitem{%
I have been in charge of hosting Prof. Vladimir Filipovic (Associate Professor at University of Belgrade, Faculty of Mathematics,
Department of Computer Science) during his sabbatical from 20/01/2018 to 30/09/2018.
 During his visit, we have collaborated on the development of metaheuristics in Bioinformatics
}
  \section{Service}

  \subsection{University Service}

  \cvitem{2021-today}{%
  \textbf{Scientific Committee Member}
  Postgraduate Master Programme ``quantitative methods for Omics Data'' (EQF
  level 8),
  University of Milano -- Bicocca
  }

\cvitem{2020-today}{%
  \textbf{Representative of the Università degli Studi di Milano -- Bicocca}
  Joint Research Unit ELIXIR IIB General Assembly,
  Italian Node of Elixir Europe, the main intergovernative Europan
  organization in Bioinformatics.
}

\cvitem{2019-today}{%
  \textbf{Deputy coordinator of the PhD program} in Computer Science,
  Univ. Milano -- Bicocca
}

\cvitem{2018-today}{%
  \textbf{Quality Assurance}, M.Sc. in Data
  Science.
}

\cvitem{2019-today}{%
      \textbf{President of the Teaching Committee (Commissione
      Didattica)}, M.Sc.
  in Data Science.
}

\cvitem{2018-today}{%
  \textbf{PI} of Università degli Studi di Milano -- Bicocca
  of the MoU (Convenzione quadro) with Istituto Nazionale di Genetica
  Molecolare (National Institute of Molecular Genetics).
}

\cvitem{2020-today}{%
  \textbf{Scientific Committee Member}, Bicocca Ambiente Società
  Economia, by decree of the rector.
}

\cvitem{2018-today}{%
      Member of the \textbf{Students-Professors Committee (Commissione
      Paritetica Docenti-Studenti)},
  Dipartimento di Informatica, Sistemistica e Comunicazione
  }\cvitem{2013-today}{%
      \textbf{Board Member}, PhD program in Computer Science,
  Univ. Milano -- Bicocca
  }

  \cvitem{2011-today}{%
      \textbf{Executive Committee (Comitato direttivo)} Member,
      Centro di Produzione Multimediale di Ateneo (Multimedia Production
      Center),
      by decree of the Academic Senate.
    }

\cvitem{2015--2020}{%
  Local Technical Coordinator,
  Univ. Milano -- Bicocca,
  Elixir IIB
  }

\cvitem{2016-today}{%
  \textbf{In charge of Elixir IIB training activities} in Univ. Milano
  -- Bicocca.
  I have organized course on ``Genome Assembly and Annotation'', ``Data
  Carpentry Workshop'', ``Software Carpentry Workshop'', ``Docker
  Advanced'', and ``Exome analysis with Galaxy''.
  }
  \cvitem{2002--2012}{%
  \textbf{Representative of the Computer Science Area},
  School of Statistics.
  }

  \cvitem{2004--2012}{%
  \textbf{Representative of the School of Statistics},
  University Committee on Computer Science.
  }

  \cvitem{2010--2012}{%
  \textbf{Delegate of the Dean on e-learning}
  School of Statistics.
  }

  \cvitem{2007--2012}{%
  \textbf{In charge of the Computer Labs}
  School of Statistics.
  }

  \cvitem{2010--2012}{%
  Member of the e-learning committee,
  School of Statistics.
  }

  \cvitem{2002--2004}{%
  \textbf{Representative of the Computer Science Area},
  School of Statistics Committee in charge of designing the B.Sc.
  program in Statistics
  (Statistica e Gestione delle Informazioni).
  The program is still active.
  }\cvitem{2004}{%
      \textbf{Representative} of the School of Statistics for the short
      course
  ``Information Technology For Problem Solving (IT4PS)'', organized by
  the CRUI Foundation.
  }\cvitem{2003}{%
  \textbf{In charge} of the course 167388 ``Laboratorio Complementare di
  Informatica per Statistici (Elective Computer Science Lab for
  Statistics)'', part of the FSE 156165 project
  ``Progetto Quadro Università degli Studi di Milano -- Bicocca''.
  The course focused on intermediate Computer Science skills for
  statistics students.
  The course has been fully funded by the European Commission.
  }

  \subsection{Service for the Research Community}

\cvitem{2023}{%
  \textbf{Program Committee Chair} of Computability in Europe (CiE) 2023.
This international conference will take place in Batumi (Georgia) in July 24-28
and is the flagship conference of the Association Computability in Europe that
has more than 1000 members and focuses on computability science in all its
multidisciplinary facets.
The conference proceedings will be published in the LNCS series.
}


\cvitem{2022}{%
  \textbf{Supervisory Board Member}, PhD Summer School ``Introduction to
  Pangenomics'',
  held at the Lake Como School of Advanced Studies, July 4-8, 2022.
The school has been attended by 25 international PhD students and had 12
teachers from Europe and the US.
The Supervisory Board had 3 members who have decided the full program of the school.
  }


\cvitem{2013-today}{%
 \textbf{Program Committee Member} of:
   Computability in Europe (CiE) 2013, 2019, 2020, 2022;
  Workshop on Algorithms in Bioinformatics (WABI) 2020, 2022;
  ISCB European Conference on Computational Biology (ECCB 2019);
  Combinatorial Pattern Matching (CPM 2019);
   Symposium on String Processing and Information Retrieval (SPIRE 2017);
   Bioinformatics Open Source Conference (BOSC 2016--2021).
  }

\cvitem{2021}{%
  \textbf{Organizer}, of Pangenome Bio Hacking (PGBH) 2021 (co-organized with
  Erik Garrison, Enza Colonna, Pjotr Prins).
This online conference has taken place on December 9-10, 2021, and we have
invited 9 speakers from Europe and the US.
It has been the first world-wide conference for researchers who have a focus on pangenomics free and open source software development.
  }

\cvitem{2018-today}{%
  Executive Officer of the \textbf{Steering Committee} of the conference
  Computability in Europe.

}

  \cvitem{2021}{%
  \textbf{Chair} of the Data Structure in Bioinformatics Workshop (DSB 2021),
  February 11-12, 2021.
The workshop had 19 research talks given from international scholars.
}


\cvitem{2020}{%
  \textbf{Editor} of the proceedings of the conference Computability in Europe 2020.
The proceedings have been published in the LNCS 12098 volume, Springer-Verlag.
  }

  \cvitem{2020}{%
  \textbf{Organizer}, of the Computability in Europe special session ``Large Scale
  Bioinformatics and Computational Sciences'' (co-organized with Iman
  Hajirasouliha, Weill-Cornell Medicine, USA).
We have selected and invited four speakers who have given an invited talk.
}


\cvitem{2016--2019}{%
  \textbf{Editorial Board} member of the journal
  ``Advances in Bioinformatics''.
}

\cvitem{2018}{%
  Evaluation committee member, PhD thesis in Computer Science, Mattia
  Gastaldello, Univ. Roma la Sapienza.
}

\cvitem{2018}{%
  Evaluation committee member, PhD thesis in Computer Science, Luca
  Ferrari, Univ. Milano.
  }


\cvitem{1997-today}{%
  Reviewer for: ACM/IEEE
  Transactions on Computational Biology and Bioinformatics, Algorithmica,
  Algorithms, Bioinformatics, Briefings in Bioinformatics, Graphs and
  Combinatorics, Information Processing Letters, INFORMS J. Computing,
  Journal of Computer Science and Technology, Theoretical Computer
  Science, Theory of Computing Systems.
  }



  \cvitem{2017}{%
  Evaluation committee member, competition for an Assistant Professor position
  (valutazione comparativa per RTDb), Univ. Milano
  }

  \cvitem{2006}{%
  Evaluation committee member, competition for an Assistant Professor position
  (valutazione comparativa per ricercatore universitario), Univ. Milano
  }

  \cvitem{2004}{%
  \textbf{special issue editor}, Journal of Computer Science and
  Technology.}

  \section{Speaker}

  \cvitem{GGI Seminar Series, March 2022}{Invited by Department of Genetics, Genomics and Informatics, University of Tennessee. Title of the talk: ``Computational Aspects of Models of Evolution''}


  \cvitem{invited speaker}{Computability in Europe 2017 special session  ``Algorithmics for Biology''}

  \cvitem{speaker at ISMB 2001}{I have presented a paper as first author ---
  and the only author of my University --- to the conference Intelligent
  Systems for Molecular Biology 2001: the main Bioinformatics scientific conference (\textbf{CORE: A, Microsoft Academic: A+})}

  \section{Research and Technology Transfer}

  \subsection{Research and consulting contracts}


\cvitem{2003--2004}{%
  Consultant on the computational aspects of the project INTERREG IIIB (2000--2006) W.E.S.T. WOMEN EAST SMUGGLING
  TRAFFICKING (WP.2.2).
  \textbf{Proponent: Fondazione Ismu -- Iniziative e
  Studi sulla Multietnicità}.
  I have been the only computer scientist involved in the project.
  }

  \cvitem{2005}{%
  Consultant on the computational aspects of the project
  ``Indagine Finalizzata all'Analisi degli Effetti Prodotti dai
  Processi di Regolarizzazione dei Lavoratori Extracomunitari, con
  Particolare Riferimento al Mercato del Lavoro e all'Integrazione Sociale
  nelle Regioni Ob. 1'', funded under Misura I.2 FESR
  ``Adeguamento del Sistema di Controllo Tecnologico del Territorio'',
  PON Sicurezza 2000/2006.
  \textbf{Proponent: Fondazione Ismu -- Iniziative e
  Studi sulla Multietnicità}.
  I have been the only computer scientist involved in the project.
  }

\section{Publications}


\nocitearticle{*}
\bibliographystylearticle{unsrt}
\bibliographyarticle{article}

\nociteconference{*}
\bibliographystyleconference{unsrt}
\bibliographyconference{conference}

\nociteinbook{*}
\bibliographystyleinbook{unsrt}
\bibliographyinbook{inbook}


\nociteP{*}
\bibliographystyleP{unsrt}
\bibliographyP{preprint}

Milano, \today\\

Gianluca Della Vedova

\end{document}
